src/diffpy/srxplanar/loadimage.py

```diff
@@ -66,31 +67,43 @@ def flipImage(self, pic):
         return pic
 
     def loadImage(self, filename):
```

Comment:
let's make the function have a pep8 compliant name.  We may as well fix these when we touch the function.